Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_light_rifle.yml

```
@@ -173,3 +173,184 @@
+      right:
+      - state: inhand-right-mag
+      - state: inhand-right-stripe
+        color: "#dbdbdb"
+
+- type: entity
+  id: MagazineDragunovUranium
+  name: "magazine (7,62mmR uranium)"
+  suffix: Sunrise
+  parent: BaseMagazineDragunov
+  components:
+  - type: BallisticAmmoProvider
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)